src/Xamarin.Android.Build.Utilities/Sdks/AndroidSdkWindows.cs

```diff
@@ -15,17 +15,13 @@ class AndroidSdkWindows : AndroidSdkBase
 		const string XAMARIN_ANDROID_INSTALLER_PATH = @"SOFTWARE\Xamarin\MonoAndroid";
 		const string XAMARIN_ANDROID_INSTALLER_KEY = "PrivateAndroidSdkPath";
 
-		public override string Adb { get { return "adb.exe"; } }
-		public override string Android { get { return "android.bat"; } }
-		public override string Emulator { get { return "emulator.exe"; } }
-		public override string Monitor { get { return "monitor.bat"; } }
-		public override string ZipAlign { get { return "zipalign.exe"; } }
-		public override string JarSigner { get { return "jarsigner.exe"; } }
-		public override string KeyTool { get { return "keytool.exe"; } }
-		public override string NdkStack { get { return "ndk-stack.exe"; } }
+		public override string ZipAlign { get; protected set; } = "zipalign.exe";
```

Comment:
@jonpryor well be kinda do. The main reason is these files 

`zipalign`, `jarsigner`, `keytool`, `javac` 

are either in the java directory which we don't scan for since we rely on the "system" one. Or are in the build-tools folder which is versioned... and we don't know the version at the point where we scan for the tools. 

So for those 4 tools we should default the extension to .exe